src/NServiceBus.Core/Encryption/WireEncryptedString.cs

```diff
@@ -63,13 +51,13 @@ public EncryptedValue EncryptedValue
         /// <summary>
         /// Only kept for backwards compatibility reasons
         /// </summary>
-        [ObsoleteEx(TreatAsErrorFromVersion = "6.0",RemoveInVersion = "6.0",Message = "No longer required")]
+        [ObsoleteEx(TreatAsErrorFromVersion = "6.0", RemoveInVersion = "7.0", Message = "No longer required")]
```

Comment:
I thought the idea is for us to obsolete gradually by using TreatAsErrorFromVersion one version higher than RemoveInVersion. Is my assumption wrong?